src/applications/vaos/newAppointmentFlow.js

```diff
@@ -180,8 +181,25 @@ export default {
           Sentry.captureException(error);
         }
 
-        dispatch(startDirectScheduleFlow(appointments));
-        return 'clinicChoice';
+        if (appointments) {
```

Comment:
This makes sure we have clinics to choose from after filtering them by past visited clinics